src/www/ui/template/include/foot.html.twig

```diff
@@ -4,6 +4,6 @@
    are permitted in any medium without royalty provided the copyright notice and this notice are preserved.
    This file is offered as-is, without any warranty.
 #}
-<script src="scripts/jquery-1.11.1.min.js" type="text/javascript"></script>
+<script src="scripts/jquery-3.2.0.min.js" type="text/javascript"></script>
```

Comment:
Have you tested that this change is compatible with all our functionality?

This might break something or change the behavior of some code. Please also check that the browser console does not log any new errors or exceptions.